web/modules/custom/employees/employees.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  employees.commands:
+    class: \Drupal\employees\Commands\BatchCommands
+    arguments: ['@entity_type.manager', '@logger.factory']
+    tags:
+      - { name: drush.command }
```

Comment:
Needs a new line at the end of the file.

Reply:
Fixed